### PR TITLE
hometask_lesson_18

### PR DESCRIPTION
**Pull request by Giuseppe7777**
hometask_lesson_18

### CODE REVIEW

File: languagescool/source/scss/contact_us.scss
Comment (RomanIlkiv) [inline languagescool/source/scss/contact_us.scss:1]: for imported files should be `_` at the beginning of a name

File: languagescool/source/scss/contact_us.scss [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    &__phone {
+        margin-bottom: 20px;
+        & p {
+            font-style: italic;
+            font-weight: bold;
+        }
+    }
+
+    &__address {
+        & p {
```
Comment (RomanIlkiv) [inline languagescool/source/scss/contact_us.scss:32]: `&` is not needed here

File: languagescool/source/scss/contact_us.scss [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        text-align: center;
+        font-size: 22px;
+        font-weight: bold;
+        margin-bottom: 20px;
+    }
+    &__frame {
+        width: 100%;
+        height: 400px;
+        border: 1px solid #000;
+        & iframe {
```
Comment (RomanIlkiv) [inline languagescool/source/scss/contact_us.scss:52]: the same mistake

File: languagescool/source/scss/contact_us.scss [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        margin-bottom: 50px;
+        padding: 0px 20px;
+        width: 400px;
+        font-weight: bold;
+        font-size: 18px;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate3d(-50%, -50%, 0);
+        & label {
```
Comment (RomanIlkiv) [inline languagescool/source/scss/contact_us.scss:87]: please, fix it here and everywhere in other places

`&` is not needed here

**Review by RomanIlkiv — CHANGES_REQUESTED** [review]

File: languagescool/source/scss/contact_us.scss (hunk 3 above)
Comment (Giuseppe7777) [reply languagescool/source/scss/contact_us.scss:87]: Ok

**Pull request merged**